### ISSUE TITLE
Citra crashes on launch

### ISSUE THREAD

**Issue by deadmeu** [citra-emu/citra#6847]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Affected Build(s)

Flatpak (nightly-1961)

### Description of Issue

Launching Citra via `flatpak run org.citra_emu.citra` hangs for a brief period of time then crashes with the following error:
`/usr/include/c++/12.2.0/bits/unique_ptr.h:445: typename std::add_lvalue_reference<_Tp>::type std::unique_ptr<_Tp, _Dp>::operator*() const [with _Tp = Memory::MemorySystem; _Dp = std::default_delete<Memory::MemorySystem>; typename std::add_lvalue_reference<_Tp>::type = Memory::MemorySystem&]: Assertion 'get() != pointer()' failed.`

I'm not sure but this may be a C++ dependency issue with the Flatpak.

### Expected Behavior

Citra should launch without issue.

### Reproduction Steps

1. Install Citra from [Flathub](https://flathub.org/apps/org.citra_emu.citra)
2. Launch the Citra Flatpak from the terminal via `flatpak run org.citra_emu.citra`
3. Observe the crash

### Log File

Log file (`~/.var/app/org.citra_emu.citra/data/citra-emu/log/citra_log.txt`) is empty

### System Configuration

Operating System: Arch Linux 
KDE Plasma Version: 5.27.7
KDE Frameworks Version: 5.108.0
Qt Version: 5.15.10
Kernel Version: 6.4.7-arch1-2 (64-bit)
Graphics Platform: Wayland
Processors: 24 × AMD Ryzen 9 3900X 12-Core Processor
Memory: 31.3 GiB of RAM
Graphics Processor: AMD Radeon RX 7900 XT

**Comment by HopelessSoap** [comment]
I am having the same issue; the following is the full output after trying to run Citra from the terminal:
``` Gtk-Message: 01:37:42.170: Failed to load module "canberra-gtk-module"
Gtk-Message: 01:37:42.170: Failed to load module "pk-gtk-module"
Gtk-Message: 01:37:42.170: Failed to load module "canberra-gtk-module"
Gtk-Message: 01:37:42.171: Failed to load module "pk-gtk-module"
qt.qpa.qgnomeplatform: Could not find color scheme  ""
Qt: Session management error: None of the authentication protocols specified are supported
/usr/include/c++/12.2.0/bits/unique_ptr.h:445: typename std::add_lvalue_reference<_Tp>::type std::unique_ptr<_Tp, _Dp>::operator*() const [with _Tp = Memory::MemorySystem; _Dp = std::default_delete<Memory::MemorySystem>; typename std::add_lvalue_reference<_Tp>::type = Memory::MemorySystem&]: Assertion 'get() != pointer()' failed.
[hopelesssoap@comput ~]$
```
### System Information:
OS: Fedora Linux 38 (Workstation Edition) x86_64 
Kernel: 6.4.7-200.fc38.x86_64 
DE: GNOME 44.3
CPU: AMD Ryzen 5 3600 (12) @ 3.600GHz
GPU: AMD ATI Radeon RX 6800 XT

**Comment by ivesfg1** [comment]
im having the same issue, im using flatpak, mine is as follows:

QStandardPaths: wrong permissions on runtime directory /run/user/1000, 0755 instead of 0700
Gtk-Message: 07:27:14.154: Failed to load module "canberra-gtk-module"
Gtk-Message: 07:27:14.154: Failed to load module "canberra-gtk-module"
qt.qpa.qgnomeplatform: Could not find color scheme  ""
Qt: Session management error: None of the authentication protocols specified are supported
/usr/include/c++/12.2.0/bits/unique_ptr.h:445: typename std::add_lvalue_reference<_Tp>::type std::unique_ptr<_Tp, _Dp>::operator*() const [with _Tp = Memory::MemorySystem; _Dp = std::default_delete<Memory::MemorySystem>; typename std::add_lvalue_reference<_Tp>::type = Memory::MemorySystem&]: Assertion 'get() != pointer()' failed.

OS: Ubuntu 20.04 x86_64
Kernel: 5.15.0-78-generic
DE: GNOME 3.36.8
CPU: INTEL I5 11400F @2.60ghz
GPU: NVIDIA RTX 2060 6GB

**Comment by SydMontague** [comment]
```ioctl (GFEATURE): Broken pipe
/usr/include/c++/13.1.1/bits/unique_ptr.h:453: typename std::add_lvalue_reference<_Tp>::type std::unique_ptr<_Tp, _Dp>::operator*() const [with _Tp = Memory::MemorySystem; _Dp = std::default_delete<Memory::MemorySystem>; typename std::add_lvalue_reference<_Tp>::type = Memory::MemorySystem&]: Assertion 'get() != pointer()' failed.
[   0.386218] Log <Critical> ?:?:0: Received signal 6
[   0.403481] Log <Critical> ?:?:0:  0# 0x00005565816161C5 in citra-qt
[   0.403514] Log <Critical> ?:?:0:  0# 0x00007FA72BA4FAB0 in /usr/lib/libc.so.6
[   0.403531] Log <Critical> ?:?:0:  0# 0x00007FA72BA9F26C in /usr/lib/libc.so.6
[   0.403544] Log <Critical> ?:?:0:  0# raise in /usr/lib/libc.so.6
[   0.403554] Log <Critical> ?:?:0:  0# abort in /usr/lib/libc.so.6
[   0.403582] Log <Critical> ?:?:0:  0# 0x00007FA72BCDD3B2 in /usr/lib/libstdc++.so.6
[   0.403588] Log <Critical> ?:?:0:  0# 0x000055658163DC2C in citra-qt
[   0.403591] Log <Critical> ?:?:0:  0# 0x00005565815C97FE in citra-qt
[   0.403596] Log <Critical> ?:?:0:  0# main in citra-qt
[   0.403607] Log <Critical> ?:?:0:  0# 0x00007FA72BA39850 in /usr/lib/libc.so.6
[   0.403614] Log <Critical> ?:?:0:  0# __libc_start_main in /usr/lib/libc.so.6
[   0.403618] Log <Critical> ?:?:0:  0# 0x00005565814DA285 in citra-qt
[   0.403620] Log <Critical> ?:?:0: RIP
```

**Comment by SydMontague** [comment]
I get the same error using citra-git from AUR.

```
ioctl (GFEATURE): Broken pipe
/usr/include/c++/13.1.1/bits/unique_ptr.h:453: typename std::add_lvalue_reference<_Tp>::type std::unique_ptr<_Tp, _Dp>::operator*() const [with _Tp = Memory::MemorySystem; _Dp = std::default_delete<Memory::MemorySystem>; typename std::add_lvalue_reference<_Tp>::type = Memory::MemorySystem&]: Assertion 'get() != pointer()' failed.
[   0.386218] Log <Critical> ?:?:0: Received signal 6
[   0.403481] Log <Critical> ?:?:0:  0# 0x00005565816161C5 in citra-qt
[   0.403514] Log <Critical> ?:?:0:  0# 0x00007FA72BA4FAB0 in /usr/lib/libc.so.6
[   0.403531] Log <Critical> ?:?:0:  0# 0x00007FA72BA9F26C in /usr/lib/libc.so.6
[   0.403544] Log <Critical> ?:?:0:  0# raise in /usr/lib/libc.so.6
[   0.403554] Log <Critical> ?:?:0:  0# abort in /usr/lib/libc.so.6
[   0.403582] Log <Critical> ?:?:0:  0# 0x00007FA72BCDD3B2 in /usr/lib/libstdc++.so.6
[   0.403588] Log <Critical> ?:?:0:  0# 0x000055658163DC2C in citra-qt
[   0.403591] Log <Critical> ?:?:0:  0# 0x00005565815C97FE in citra-qt
[   0.403596] Log <Critical> ?:?:0:  0# main in citra-qt
[   0.403607] Log <Critical> ?:?:0:  0# 0x00007FA72BA39850 in /usr/lib/libc.so.6
[   0.403614] Log <Critical> ?:?:0:  0# __libc_start_main in /usr/lib/libc.so.6
[   0.403618] Log <Critical> ?:?:0:  0# 0x00005565814DA285 in citra-qt
[   0.403620] Log <Critical> ?:?:0: RIP
zsh: IOT instruction (core dumped)  citra-qt
```

OS: Manjaro Linux
Kernel: 6.4.6
DE: KDE Plasma 5.27.6
CPU: Ryzen 7700X
GPU: NVidia 3060ti
Graphics Platform: X11

**Comment by HopelessSoap** [comment]
> @HopelessSoap Just to confirm are you also using the Flatpak?

Yes, I am using the Flatpak

**Comment by ivesfg1** [comment]
if someone finds a fix, please share... i was in an exciting pokemon alpha sapphire run yesterday, and got frustrated today when the emulator didn't want to open anymore XD

**Comment by GPUCode** [comment]
Fixed by https://github.com/citra-emu/citra/pull/6849

**Comment by ivesfg1** [comment]
> Fixed by #6849

didnt fixed for me

**Comment by ivesfg1** [comment]
> > Fixed by #6849
> 
> didnt fixed for me

oh nevermind, just saw it is only going to be merged in the next nightly, thx!
